src/tests/unit/tests/electron/platform/android/rule-information-provider.test.ts

```diff
@@ -14,36 +14,33 @@ describe('RuleInformationProvider', () => {
         provider = new RuleInformationProvider();
     });
 
-    function buildTouchSizeWcagRuleResultObject(
-        status: string,
-        dpi: number,
-        height: number,
-        width: number,
-        axeViewId: string = null,
-    ): RuleResultsData {
+    function buildTouchSizeWcagRuleResultObject(status: string, dpi: number, height: number, width: number): RuleResultsData {
         const props = {};
         // This is based on the output of the Android service
         props['Screen Dots Per Inch'] = dpi;
         props['height'] = height;
         props['width'] = width;
 
-        return buildRuleResultObject('TouchSizeWcag', status, axeViewId, props);
+        return buildRuleResultObject('TouchSizeWcag', status, null, props);
     }
 
     function buildColorContrastRuleResultObject(
         status: string,
         ratio: number,
         foreground: string,
         background: string,
-        axeViewId: string = null,
```

Comment:
axeViewId should have been removed when RuleInformationProvider was factored out. Fixing the oversight now.